coherence-spring-docs/src/main/asciidoc/spring-data.adoc

```diff
@@ -217,4 +218,224 @@ import org.springframework.scheduling.annotation.EnableAsync;
 @EnableCoherenceRepositories
 public static class Config {
 }
+----
+
+== Projections
+
+Spring Data Coherence module supports projections as defined in {spring-docs-data-projections}[Spring Data Projections] documentation. This allows us
+(among other things) to transfer over network just requested properties.
```

Comment:
Perhaps reword to "This allows us, among other things, to transfer a subset of the entities properties."  ?